publications/tasks.py

```diff
@@ -1,27 +1,73 @@
 from django_q.models import Schedule
-from datetime import date, timezone
 from publications.models import Publication
-from django.core.mail import send_mail
+from views import harvest_data
+import requests
+from bs4 import BeautifulSoup
+import json
+import xml.dom.minidom
+from xml.dom.minidom import parseString
+from django.db.models import Max
+from django.contrib.gis.geos import GEOSGeometry
+
+def harvest_demo():
+    url1 = 'https://service.tib.eu/optimeta/index.php/optimeta/oai/?verb=ListRecords&metadataPrefix=oai_dc'
+    harvest_data(url1)
+
+def get_geom(url):
+    #soup = BeautifulSoup(open(url,encoding="utf8"), "html.parser") # forlocal file
+    req= requests.get(url)
+    soup = BeautifulSoup(req.content, 'html.parser')
+    for tag in soup.find_all("meta"):
+        if tag.get("name", None) == "DC.SpatialCoverage":
```

Comment:
What does this function return, when the meta tag with that name is not found?

Maybe `None` ?

We need a test case for that possibility!

Reply:
Added a test case where the function returns none in new PR Harvesting data from OJS #76